analysers/analyser_merge_parking_FR_BNLS.py

```diff
@@ -0,0 +1,110 @@
+            fix = T_(
+'''If you're sure that it's a new data for OpenStreetMap, then you can add it with your favorite editor (picture of Id, josm link ?)'''),
+            trap = T_(
+'''It's not because it's open that it's right,see it's because it's opendata that OSM contributors needs to check '''),
+            doc3 = dict(
+            title = T_('Integration suggestion'),
+            detail = T_(
+''''This is a integration suggestion, mixing opendatabase and OpenStreetMap.'''),
+            fix = T_(
+'''If you're sure that it's a good integration then you can add it with your favorite editor (picture of Id, josm link ?)'''),
+            trap = T_(
+'''It's not because there's a integration suggestion that it's right, you're OSM contributors and not a machine !'''),
+            doc4 = dict(
+            title = T_('Update'),
+            detail = T_(
+''''This is an update suggestion because there's the same ref in opendatabase and OSM.'''),
+            fix = T_(
+'''If you're sure that it's a good update then you can add it with your favorite editor (picture of Id, josm link ?)'''),
+            trap = T_(
+'''It's not because there's a update suggestion that it's right, you're OSM contributors and not a machine !''')
+        self.missing_official = self.def_class(item = 8130, id = 1, level = 3, tags = ['merge', 'parking'],**self.merge_docs(doc1,
+            title = T_f('{0} parking not integrated', 'BNLS'),
+            detail = T_(
+'''This parking is referenced in the database of car parks managed by local authorities in France, off-street.
+This database is made up of local authorities and is available on data.gouv.fr.'''),
+            fix = T_(
+'''See [the mapping](https://wiki.openstreetmap.org/wiki/France/data.gouv.fr/Base_nationale_des_lieux_de_stationnement)
+on the  wiki. Add a node or add tags if already existing.'''),
+            trap = T_(
+'''lorem ipsum trap not integrated'''))),
+        self.possible_merge = self.def_class(item = 8131, id = 3, level = 3, tags = ['merge', 'parking'],**self.merge_docs(doc3,
+            title = T_f('{0} parking integration suggestion', 'BNLS'),
+            detail = T_(
+'''lorem ipsum specific explanation for integration suggestion for specific opendata base'''),
+            fix = T_(
+'''lorem ipsum, How to fix for integration suggestion for specific opendata base'''),
+            trap = T_(
+'''lorem ipsum trap for integration suggestion for specific opendata base'''))),
+        self.update_official = self.def_class(item = 8132, id = 4, level = 3, tags = ['merge', 'parking'],**self.merge_docs(doc4,
+            title = T_f('{0} parking  update', 'BNLS'))
+            detail = T_(
+'''lorem ipsum specific explanation about update for specific opendata base'''),
+            fix = T_(
+'''lorem ipsum, How to fix about update for specific opendata base'''),
+            trap = T_(
+'''lorem ipsum trap about update for specific opendata base'''))),
```

Comment:
```suggestion
            doc1 = dict(
                title = T_('Opendata'),
                detail = T_(
''''This is a data from an opendatabase, without any verification'''),
                fix = T_(
'''If you're sure that it's a new data for OpenStreetMap, then you can add it with your favorite editor (picture of Id, josm link ?)'''),
                trap = T_(
'''It's not because it's open that it's right,see it's because it's opendata that OSM contributors needs to check '''),
            doc3 = dict(
                title = T_('Integration suggestion'),
                detail = T_(
''''This is a integration suggestion, mixing opendatabase and OpenStreetMap.'''),
                fix = T_(
'''If you're sure that it's a good integration then you can add it with your favorite editor (picture of Id, josm link ?)'''),
                trap = T_(
'''It's not because there's a integration suggestion that it's right, you're OSM contributors and not a machine !'''),
            doc4 = dict(
                title = T_('Update'),
                detail = T_(
''''This is an update suggestion because there's the same ref in opendatabase and OSM.'''),
                fix = T_(
'''If you're sure that it's a good update then you can add it with your favorite editor (picture of Id, josm link ?)'''),
                trap = T_(
'''It's not because there's a update suggestion that it's right, you're OSM contributors and not a machine !''')
        self.missing_official = self.def_class(item = 8130, id = 1, level = 3, tags = ['merge', 'parking'],**self.merge_docs(doc1,
            title = T_f('{0} parking not integrated', 'BNLS'),
            detail = T_(
'''This parking is referenced in the database of car parks managed by local authorities in France, off-street.
This database is made up of local authorities and is available on data.gouv.fr.'''),
            fix = T_(
'''See [the mapping](https://wiki.openstreetmap.org/wiki/France/data.gouv.fr/Base_nationale_des_lieux_de_stationnement)
on the  wiki. Add a node or add tags if already existing.'''),
            trap = T_(
'''lorem ipsum trap not integrated'''))),
        self.possible_merge = self.def_class(item = 8131, id = 3, level = 3, tags = ['merge', 'parking'],**self.merge_docs(doc3,
            title = T_f('{0} parking integration suggestion', 'BNLS'),
            detail = T_(
'''lorem ipsum specific explanation for integration suggestion for specific opendata base'''),
            fix = T_(
'''lorem ipsum, How to fix for integration suggestion for specific opendata base'''),
            trap = T_(
'''lorem ipsum trap for integration suggestion for specific opendata base'''))),
        self.update_official = self.def_class(item = 8132, id = 4, level = 3, tags = ['merge', 'parking'],**self.merge_docs(doc4,
            title = T_f('{0} parking  update', 'BNLS'))
            detail = T_(
'''lorem ipsum specific explanation about update for specific opendata base'''),
            fix = T_(
'''lorem ipsum, How to fix about update for specific opendata base'''),
            trap = T_(
'''lorem ipsum trap about update for specific opendata base'''))),
```